docs/guides/chip_tool_guide.md

```diff
@@ -0,0 +1,769 @@
+-   Closing the Bluetooth LE connection, as the commissioning process is
+    finished and the `chip-tool` is now using only the IPv6 traffic to reach the
+    device.
+
+#### Commissioning over IP
+
+The command below will discover devices and try to pair with the first one it
+discovers using the provided _setup PIN code_.
+
+```
+./BUILD_PATH/chip-tool pairing onnetwork <node_id> <pin_code>
+
+```
+
+where:
+
+-   _<node_id\>_ is the user-defined ID of the node being commissioned,
+-   _<pin_code\>_ is device specific _setup PIN code_ determined in step 4
+
+The command below will discover devices with long discriminator and try to pair
+with the first one it discovers using the provided setup code.
+
+```
+./BUILD_PATH/chip-tool pairing onnetwork-long <node_id> <pin_code> <discriminator>
+
+```
+
+where:
+
+-   _<node_id\>_ is the user-defined ID of the node being commissioned,
+-   _<pin_code\>_ and _<discriminator\>_ are device specific keys determined in
+    step 4
+
+The command below will discover devices based on the given QR code payload
+(which devices log when they start up) and try to pair with the first one it
+discovers.
+
+```
+./BUILD_PATH/chip-tool qrcode <node_id> <qrcode_payload>
+
+```
+
+where:
+
+-   _<node_id\>_ is the user-defined ID of the node being commissioned,
+-   _<qrcode_payload\>_ is the QR code payload
+
+#### Forget the currently-commissioned device
+
+```
+./BUILD_PATH/chip-tool pairing unpair <node_id>
+
+```
+
+where:
+
+-   _<node_id\>_ is the user-defined ID of the node which is going to be forgot
+    by the `chip-tool`
+
+#### Note
+
+`chip-tool` currently only supports commissioning and remembering one device at
+a time. The configuration state is cached in `/tmp/chip_tool_config.ini`;
+deleting this and other .ini files in `/tmp` can sometimes resolve issues
+related to stale configuration.
+
+### Step 6: Control application ZCL clusters.
+
+For the lighting-app example, execute the following command to toggle the LED
+state:
+
+```
+./BUILD_PATH/chip-tool onoff toggle <node_id> <endpoint_id>
+```
+
+where:
+
+-   _<node_id\>_ is the user-defined ID of the commissioned node
+-   _<endpoint_id\>_ is the ID of the endpoint with OnOff cluster implemented
+
+To change the brightness of the LED, use the following command, with the
+_<level\>_ equal to value between 0 and 255.
+
+```
+./BUILD_PATH/chip-tool levelcontrol move-to-level <level> <transition_time> <option_mask> <option_override> <node_id> <endpoint_id>
+```
+
+where:
+
+-   _<level\>_ is the brightness level encoded between 0 and 255
+-   _<transition_time\>_ is the transition time
+-   _<option_mask\>_ is the option mask
+-   _<option_override\>_ is the option override
+-   _<node_id\>_ is the user-defined ID of the commissioned node
+-   _<endpoint_id\>_ is the ID of the endpoint with LevelControl cluster
+    implemented
+
+In case of doubts regarding any of the options, please refer to the Matter
+specification.
```

Comment:
Which is not yet public, so we can't advertise it like this. Even if we could, we would need to provide at least the number of the section.